### DISCUSSION TITLE
LLVM-C.dll

### DISCUSSION THREAD

**Discussion by Sapta0802** [NVlabs/sionna#453] [General]
![image](https://github.com/NVlabs/sionna/assets/88283648/745b10bc-1443-4d63-b1aa-8cbf97b38f63)

AttributeError: jit_init_thread_state(): the LLVM backend is inactive because the LLVM shared library ("LLVM-C.dll") could not be found! Set the DRJIT_LIBLLVM_PATH environment variable to specify its path

1. Downloaded LLVM  file for window 
2. change the environment variable path to LLVM bin path 

![image](https://github.com/NVlabs/sionna/assets/88283648/a33923f9-e7dc-4fe2-8db2-789988fedfbd)
![image](https://github.com/NVlabs/sionna/assets/88283648/85d1b735-dce0-4c19-b2b8-a9501f20121c)

but the problem was not fixed, can you please help me to fix this problem in Windows 11.

**Comment by merlinND** [comment]
Hello @Sapta0802,

This is an issue related to DrJit. Please see this thread with existing solutions: https://github.com/mitsuba-renderer/drjit/issues/196

If it doesn't fix it, could you please open an issue with a detailed description of your problem and the steps you took to fix it (including precisely how did you install LLVM) in the DrJit repository?
https://github.com/mitsuba-renderer/drjit/issues

**Reply by Sapta0802** [reply]
thanks a lot, this problem is fixed.
